actionpack/lib/action_controller/metal/implicit_render.rb

```diff
@@ -2,8 +2,14 @@ module ActionController
   module ImplicitRender
     def send_action(method, *args)
       ret = super
-      default_render unless performed?
-      ret
+      return ret if performed?
```

Comment:
@georgeclaghorn I liked your suggestion (maybe deleted because it's not here?) but get 593 test failures when implemented. Not sure what's going on there